doc/tech/roadmap.md

```diff
@@ -61,10 +61,51 @@ Please refer to individual [release plan](./release_notes/) and features for det
 | Release Candidate | RC0 | The goal of the Release Candidate is to ensure the documentations are properly aligned, fully reviewed in the new release branch. | Prioritise the required fixes and address them. If there are any critical fixes required then the fixes will be provided and it will be tagged with minor release. (Eg. Baldy 4.0.1). | |
 | Release Sign-Off | | The goal of the Release Sign-Off review is to ensure all the projects are successfully passed all the review. All the committed deliverables are available and passed the quality criteria. | | |
 
-
+<p align="center"><b>Table 1:</b> Detailed Milestones</p>
 
 <a name="3"></a>
 ## Dependencies between various Workstreams
 
+The various workstreams in CNTT are 
+- Reference Model (RM)
+- Reference Architecture (RA)
+- Reference Implementation (RI) 
+- Reference Conformance (RC)
+
+The workstream dependency relationship in simple terms, [Reference Conformance](../ref_cert) verifies and tests the [Reference Implementation](../ref_impl) which follows the requirements and architecture defined in the CNTT [Reference Architecture](../ref_arch) and Reference Architecture describes the high level system components and its interactions by adhering to the requirements and expectations set by the CNTT [Reference Model](../ref_model) which sets the standards for infrastructure abstraction, compliance and verification. 
+
+For the standard release stabilisation, the pattern in each release follows RM must be the advanced version of RA. Similarly RA must be the advanced version of RI and simultaneously for RC.
+
+For the standard release stabilisation, each release follows the pattern of one workstream is functioning ahead of another workstream in the sequence of Reference Model, Reference Architecture, Reference Implementation and Reference Conformance. For eg. In Baldy release, RM3.0 inputs will feed into RA2.0 and likewise as shown in the Figure 3.
+
+There are two different tracks in CNTT
+- Virtualized workloads, deployed on OpenStack
+- Containerized workloads, deployed on Kubernetes
+
+Each track follows the industry driven standards in the Reference Model as depicted in the below diagram.
+
+
+<p align="center"><img src="./figures/CNTT_Stable_3.0.png" alt="CNTT Stable 3.0 Portrait" title="CNTT Stable 3.0 Portrait
+" width="100%"/>
+ <p align="center"><b>Figure 3:</b> CNTT Stable 3.0 Portrait</p>
+
+
 <a name="4"></a>
 ## Dependencies with Industry Communities
+
+The CNTT is collaboratively working with other standard bodies within the Open Source industry. Some of the projects are 
+- OpenStack
+- Kubernetes
+- OPNFV
+- ONAP
+- CNCF
+- ESTI NFV ISG
+
```

Comment:
K8s under CNCF , So we need to remove K8s

and typo ESTI NFV ISG ==> ETSI NFV ISG

Reply:
@ASawwaf - Thanks. Typo corrected now. I didn't get your first point

Reply:
@prabhubalan ,kubernetes project is a project under CNCF , no need to put Kubernetes and CNCF , we can remove Kubernetes , what did you think

Reply:
@ASawwaf - I agree. Removed it now.